ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -214,6 +228,16 @@ jobs:
     params:
       file: geode-build-version/number
 {% endfor %}
+- name: ColumnSeparator
+  serial: false
+  plan:
+  - get: geode
+    trigger: true
+    passed: [Build]
+  - get: geode-build-version
+    params:
+      pre: ((semver-prerelease-token))
```

Comment:
Using the `pre` parameter on semver causes the build ID to rev. This is bad. We only want the version to be changed once in the pipeline, by the `Build` job. The parameter should be removed from this instance. See [https://github.com/concourse/semver-resource#version-bumping-semantics](https://github.com/concourse/semver-resource#version-bumping-semantics
).

Reply:
good catch.  fixed anyway, even though this PR seems unlikely to ever get merged.